user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md

```diff
@@ -0,0 +1,35 @@
+---
+uid: KI_Cube_connection_issue_alarm_tabs
+---
+
+# Cube freezes on 'Connected!' loading screen when no alarm tabs are displayed
+
+## Affected versions
+
+DataMiner versions 10.3.9 and 10.3.10.
+
+## Cause
+
+By default, the [three default alarm tabs](xref:ChangingTheAlarmConsoleLayout#the-three-default-alarm-tabs) are displayed at startup: active alarms, information events, and masked alarms. However, when the Alarm Console settings have been configured to display no tabs in the Alarm Console, a startup issue may occur.
```

Comment:
"Startup issue" usually implies a server startup issue. In this case, "a connection issue" is more accurate.